src/MarketingPageAcceptanceTestsSpecflow/Steps/ClientApplication/EditClientApplicationType.cs

```diff
@@ -15,12 +15,13 @@ public EditClientApplicationType(UITest test, ScenarioContext context) : base(te
         {
         }
 
-        [Given(@"that a Client Application Type is selected")]
-        public void GivenThatAClientApplicationTypeIsSelected()
+        [Given(@"that a (.*) Client Application Type is selected")]
```

Comment:
Could change this to be a fixed selection of permittable inputs rather than a catch all?